server/tests/test_token.py

```diff
@@ -0,0 +1,54 @@
+# Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+import base64
+import hashlib
+
+from parsec._parsec import DateTime, SecretKey
+from parsec.components.auth import AccountAuthenticationToken, AccountAuthenticationType
+
+
+def gen_account_auth_token() -> tuple[AccountAuthenticationToken, SecretKey, bytes]:
+    email = "foo@example.com"
+    timestamp = DateTime.from_timestamp_seconds(1744721639)
```

Comment:
It's a specific value ?

Reply:
It was the timestamp at when I written the test, but it does not hold a particular value

Reply:
It would be nice to add a comment, it does not cost much and may save time for our future selves.
```suggestion
    // Timestamp from the moment the test was written, does not hold a particular value 
    timestamp = DateTime.from_timestamp_seconds(1744721639)
```
Or just use a plain readable date as with the other dummy values :)
```suggestion
    timestamp = DateTime.from_ymd_hms_us(2025, ...)
```

Reply:
Going with the second suggestion